doc/internal/release/how-to-release.md

```diff
@@ -77,19 +78,15 @@ That includes:
   > - While the Vitess Operator is located in a different repository, we also need to do a release for it.
   > - The Operator follows the same cycle: RC1 -> GA -> Patches.
   > - Documentation for the pre-release of the Vitess Operator is available [here](https://github.com/planetscale/vitess-operator/blob/main/docs/release-process.md#prepare-for-release).
-- **Update the release notes on `main`.**
-  > - One Pull Request against `main` must be created, it will contain the new release notes that we are adding in the Release Pull Request.
-  > - We open this Pull Request now to avoid waiting on the CI during release day.
-  > - All future changes to the release notes during the code freeze will need to be ported to both PRs: the one on `main` and the Release Pull Request.
 - **Update the website documentation.**
   > - We want to open a preparatory **draft** Pull Request to update the documentation.
   > - There are several pages we want to update:
-      >   - [The releases page](https://vitess.io/docs/releases/), we must add the new release to the list with all its information and link. The links can be broken (404 error) while we are preparing for the release, this is fine.
-  >   - [The local install page](https://vitess.io/docs/get-started/local/), we must use the proper version increment for this guide and the proper SHA. The SHA will have to be modified once the Release Pull Request and the release is tagged is merged.
+  >  - [The releases page](https://vitess.io/docs/releases/), we must add the new release to the list with all its information and link. The links can be broken (404 error) while we are preparing for the release, this is fine.
+  >  - [The local install page](https://vitess.io/docs/get-started/local/), we must use the proper version increment for this guide and the proper SHA. The SHA will have to be modified once the Release Pull Request and the release is tagged is merged.
   > - If we are doing a GA or RC release follow the instructions below:
-      >  - There are two scripts in the website repository in `./tools/{ga|rc}_release.sh`, use them to update the website documentation. The scripts automate:
-           >    - For an RC, we need to create a new version in the sidebar and mark the current version as RC.
-  >    - For a GA, we need to mark the version we are releasing as "Stable" and the next one as "Development".
+  >  - There are two scripts in the website repository in `./tools/{ga|rc}_release.sh`, use them to update the website documentation. The scripts automate:
+  >  - For an RC, we need to create a new version in the sidebar and mark the current version as RC.
```

Comment:
```suggestion
  >  - For an RC, we need to create a new version in the sidebar and mark it as RC.
```

Reply:
Changing `the current version` to `it` would imply that we are changing the new version in the sidebar to `RC`, which is incorrect. Let's assume we are releasing `v18.0.0-rc1`. We want to create a new version in the sidebar (`v19.0`), and mark the current version (the one we are releasing, `v18.0`) as `RC`. I will change the wording to make it more clear :)

Reply:
This is fixed via 136c780568